mmv1/products/privateca/CaPool.yaml

```diff
@@ -471,6 +471,16 @@ properties:
           in all issued Certificates. If this is false, CRLs will not be published and the corresponding X.509 extension will not
           be written in issued certificates. CRLs will expire 7 days from their creation. However, we will rebuild daily. CRLs are
           also rebuilt shortly after a certificate is revoked.
+      - !ruby/object:Api::Type::Enum
```

Comment:
Something to think about with Enum fields is whether the set of values will change frequently over time - if so, we generally try to use string fields and leave validation up to the API. Otherwise you can end up in a situation where someone is forced to upgrade the provider solely to get the validation to allow a new value.

Reply:
Thanks for sharing and it make sense.

This field `encoding_format` will be very stable regarding its candidate value. So, it should be fine to use `Enum` here.